tests/gmsh/create_tria_01.output.gmsh-4

```diff
@@ -0,0 +1,40 @@
+
+DEAL::  1 active cells
```

Comment:
In this test, we let `Gmsh` create a `msh` file `geo` geometry script, parse it and create a `Triangulation` object from it. In the end, we just print the number of cells created, some kind of hash created from the vertices and the `msh` file.
Since version 4 of the MSH file format differs from version 2, the created `msh` file is different and we need an alternative output file.